features/in_onderzoek.feature

```diff
@@ -597,6 +597,173 @@ Functionaliteit: in onderzoek
+      | verblijfadres.inOnderzoek.adresregel3                      | true            |
+      | verblijfadres.inOnderzoek.land                             | true            |
+      | inOnderzoek.datumIngangOnderzoek.datum                     | 2022-03-07      |
+      | inOnderzoek.datumIngangOnderzoek.type                      | Datum           |
+      | inOnderzoek.datumIngangOnderzoek.langFormaat               | 7 maart 2022    |
+      | inOnderzoek.type                                           | true            |
+      | inOnderzoek.datumVan                                       | true            |
+
+    Scenario: het element land van de groep adres buitenland van de categorie verblijfplaats is in onderzoek
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                        | waarde    |
+      | burgerservicenummer         | 555550001 |
+      En de persoon heeft GEEN 'persoonInOnderzoek' gegevens
+      En persoon met burgerservicenummer '555550001' heeft de volgende 'verblijfplaats' gegevens
+      | naam        | waarde          |
+      | land        | 5010            |
+      | adresregel1 | Rue du pomme 25 |
+      | adresregel2 | Bruxelles       |
+      En persoon met burgerservicenummer '555550001' heeft de volgende 'verblijfplaats.inOnderzoek' gegevens
+      | naam                          | waarde   |
+      | aanduidingGegevensInOnderzoek | 081310   |
+      | datumIngangOnderzoek          | 20220307 |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                             |
+      | type                | RaadpleegMetBurgerservicenummer    |
+      | burgerservicenummer | 555550001                          |
+      | fields              | burgerservicenummer,verblijfplaats |
+      Dan heeft het kind met burgerservicenummer '555550002' de volgende 'verblijfplaats' gegevens
+      | naam                                                       | waarde          |
+      | verblijfadres.adresregel1                                  | Rue du pomme 25 |
+      | verblijfadres.adresregel2                                  | Bruxelles       |
+      | verblijfadres.land                                         | 5010            |
+      | verblijfadres.inOnderzoek.datumIngangOnderzoek.datum       | 2022-03-07      |
+      | verblijfadres.inOnderzoek.datumIngangOnderzoek.type        | Datum           |
+      | verblijfadres.inOnderzoek.datumIngangOnderzoek.langFormaat | 7 maart 2022    |
+      | verblijfadres.inOnderzoek.land                             | true            |
+      | inOnderzoek.datumIngangOnderzoek.datum                     | 2022-03-07      |
+      | inOnderzoek.datumIngangOnderzoek.type                      | Datum           |
+      | inOnderzoek.datumIngangOnderzoek.langFormaat               | 7 maart 2022    |
+      | inOnderzoek.type                                           | true            |
```

Comment:
waarom is hier het type in onderzoek?

Reply:
ik vind in de feature geen aparte rule die beschrijft wanneer type (voor polymorfie) in onderzoek wordt gezet. Dus moet dat worden uitgeleid uit Rule: een afgeleid gegeven wordt in inOnderzoek opgenomen wanneer ten minste één van de gegevens waaruit het wordt afgeleid in onderzoek staat.

Deze rule staat op regel 881, dus later dan wat hier wordt getoond. Dan kan je het ook niet begrijpen. Ik denk dat je iets moet doen aan de volgorde, of aan de keuze van scenario's

Aparte regels of scenario's voor bepalen/tonen wanneer type in onderzoek wordt gezet (voor nationaliteit en voor verblijfplaats) is wellicht nuttig. Nu moet de gebruiker zelf de regel 881 combineren met wat in verblijfplaats.feature staat over afleiden van type. Dat gaat niemand doen, behalve de provider (en dan maar hopen dat onze provider developer @MelvLee begrijpt hoe dat moet).

Reply:
We laten het mergen van dit feature niet afhangen van het herordenen en splitsen van dit feature. Hiervoor maak ik daarom een apart issue aan.